c7n/resources/networkmanager.py

```diff
@@ -46,8 +46,59 @@ class resource_type(TypeInfo):
 GlobalNetwork.filter_registry.register('marked-for-op', TagActionFilter)
 
 
+@c7n_resources.register('networkmanager-link')
+class Link(ChildResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'networkmanager'
+        enum_spec = ('get_links', 'Links', None)
+        parent_spec = ('networkmanager-global', 'GlobalNetworkId', None)
+        arn = 'LinkArn'
+        name = 'LinkId'
+        id = 'LinkId'
+        date = 'CreatedAt'
+        config_type = 'AWS::NetworkManager::Link'
+        cfn_type = 'AWS::NetworkManager::Link'
+        permissions_augment = ("networkmanager:ListTagsForResource",)
```

Comment:
there isn't any augment though re this permission being listed on the resources?

Reply:
@kapilt Removed the augments